package.json

```diff
@@ -140,7 +154,7 @@
     "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^11.0.4",
     "@testing-library/react-hooks": "^8.0.1",
-    "babel-jest": "^26.3.0",
+    "babel-jest": "^29.5.0",
```

Comment:
Probably can be removed, it's also in `jest-config-stripes`. And the same for `jest` and `jest-environment-jsdom`.

Reply:
Different versions of jest cause errors, so we need to upgrade to the same as in `jest-config-stripes`

Reply:
I assume the complete removal of the `babel-jest`, `jest` and `jest-environment-jsdom` packages shouldn't break anything because there are already exist with the same version in `jest-config-stripes`.